src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -87,7 +89,10 @@ export const NavBar: React.FC = track(
   return (
     <>
       <NavBarSkipLink />
```

Comment:
We want the skip link to be the first focusable thing. This is a little hard to enforce unfortunately.